src/array.ts

```diff
@@ -83,17 +84,23 @@ export function from<T>(arrayLike: (string | ArrayLike<T>), mapFunction?: MapCal
 	}
 
 	const Constructor: any = this;
```

Comment:
I will admit, I am not sure either, it could easily have been some code "rot".  I think we can merge things and open another ticket to work on trying to clean this up a bit.